java401_code_challenges/src/main/java/java401_code_challenges/hashtable/HashTable.java

```diff
@@ -2,31 +2,31 @@
 
 public class HashTable<T>{
 
-    private int bucketSize = 100;
+    private int numberOfBuckets = 100;
 
-    public Linkedlist[] bucket = new Linkedlist[bucketSize];
+    public Linkedlist[] bucketsArray = new Linkedlist[numberOfBuckets];
```

Comment:
Your linked list is a generic class, but you did not specify a type. Always be specific!